react/helpers/tracker.spec.js

```diff
@@ -0,0 +1,36 @@
+import * as trackerFile from './tracker'
+
+class Tracker {
+  constructor() {}
+  push(event) {
+    return event
+  }
+}
+
+describe('tracker / piwik action', () => {
+  let mocked = new Tracker()
+  let getTrackerSpy, pushSpy
+  beforeEach(() => {
+    getTrackerSpy = jest.spyOn(trackerFile, 'getTracker')
+    getTrackerSpy.mockImplementation(() => mocked)
```

Comment:
IMHO, it'd maybe be simpler to remove the `Tracker` class and to do 

`getTrackerSpy.mockImplementation(() => ({ push: jest.fn() }))` ?